_locales/en/messages.json

```diff
@@ -26,14 +26,17 @@
   "1hour": {
     "message": "For 1 hour"
   },
+  "4hours": {
+    "message": "For 4 hours"
+  },
   "8hours": {
     "message": "For 8 hours"
   },
-  "24hours": {
-    "message": "For 24 hours"
+  "untilTomorrow": {
+    "message": "Until tomorrow"
   },
   "untilEnabled": {
-    "message": "Until I turn it back on"
+    "message": "Until I turn it off"
```

Comment:
Discord uses `Until I turn it back on`, do we actually know better than them?

Reply:
![image](https://github.com/user-attachments/assets/fc9c22a3-ee99-447d-802c-4dd66b5a1019)

Reply:
In Discord it makes sense because "it" = "server". The context in our UI is different, so I think something shorter like "Until unmuted" would be better.

Reply:
> In Discord it makes sense because "it" = "server". The context in our UI is different, so I think something shorter like "Until unmuted" would be better.

We've replaced the term "mute" with "do not disturb", but we could say "until turned off".

Reply:
```suggestion
    "message": "Until turned off"
```